### DISCUSSION TITLE
Release v0.2.0 is here!

### DISCUSSION THREAD

**Discussion by Mixss** [esoviscode/discord-bot#120] [Announcements]
Hello everyone,
The amount of work we have put into this project over the last month has been enormous. I'm very happy with how it's turned out, and I hope we can continue to increase our productivity in order to deliver the best possible game master bot.
See you in development!

**Comment by pcichowski** [comment]
@Mixss  Could you tell us what the main accomplishments of this release are? 🤗

**Comment by Mixss** [comment]
The game is playable with correct graphical interface and players' turns
